content/python/concepts/files/terms/truncate/truncate.md

```diff
@@ -36,3 +36,22 @@ f = open("gullivers_travels.txt", "r")
 
 print(f.read())
 ```
+## Codebyte Example
```

Comment:
there should be one empty line between the previous code block and the codebyte header